pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -112,14 +131,19 @@ var (
 
 	caRSAKeySize = env.RegisterIntVar("CITADEL_SELF_SIGNED_CA_RSA_KEY_SIZE", 2048,
 		"Specify the RSA key size to use for self-signed Istio CA certificates.")
-)
 
-type CAOptions struct {
-	// domain to use in SPIFFE identity URLs
-	TrustDomain    string
-	Namespace      string
-	Authenticators []authenticate.Authenticator
-}
+	//TODO: Likely to be removed and added to mesh config
+	externalCA = env.RegisterStringVar("EXTERNAL_CA", "",
+		"External CA Type").Get()
+
+	//TODO: Likely to be removed and added to mesh config
+	k8sSigner = env.RegisterStringVar("K8S_SIGNER", "",
+		"Kubernates CA Signer type. Valid from Kubernates 1.18").Get()
+
+	//TODO: Likely to be removed and added to mesh config
+	externalCACert = env.RegisterStringVar("EXT_CA_ROOT_PATH", defaultCACertPath,
```

Comment:
How to mount the real CA cert into sidecar?

Reply:
Not sure why the real CA Cert needs to be mounted into the sidecar for this particular use case? Correct me if I am wrong; the workflow so far has been - Istiod ensures that the certificate has been signed by a CA (corresponding to the CA Cert mounted), appends the CA Cert to the cert-chain and forwards this cert-chain back to the sidecar. The sidecar does not do any verification, it merely checks if the root CA of the signed cert is different from what is already mounted and updates its secret with the new Root CA.

Reply:
I mean where is the external ca-cert. How do you mount it to istiod?

Reply:
Will be fixing this in the next PR/next few commits. external-ca-cert will be a secret that will be mounted onto ISTIOD that contains root-cert.pem (the CA cert)

Reply:
Changing this in the latest PR. File location will be fixed as /etc/external-ca-cert